docs/Submit_a_new_ontology.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: doc
-id: Policy_for_OBO_namespace_and_associated_PURL_requests
-title: Policy for OBO IDSPACE and associated PURL requests
+id: Submit_a_new_ontology
```

Comment:
I think we should change this (and the title) to be something like "OBO Foundry membership requirements and technical details".

Reply:
i made this change